woocommerce/woocommerce-gutenberg-products-block
# Mini Cart Contents Block: Add text and link color support

This PR adds text and link color support for the Mini Cart Contents Block.



Fixes #5413



### Screenshots

![image](https://user-images.githubusercontent.com/4463174/147122828-58b497d6-dff5-447b-b155-ae75b6f382fa.png)
*editor side*

![image](https://user-images.githubusercontent.com/4463174/147122905-e3688a3c-f360-4e17-97a7-5567eb340614.png)
*frontend side*

### Testing

### Manual Testing

How to test the changes in this Pull Request:

Check out this branch

1. Open the FSE editor.
2. Pick the Mini Cart Content templates.
3. Be sure to focus on the `Mini Cart Contents`.
4. On the right sidebar, set different colors.
5. Check if the new colors are visible on the editor and frontend side.


### Question
I'm not sure that this is PR is complete.

I have some questions about both statuses.

#### Filled Cart

![image](https://user-images.githubusercontent.com/4463174/147125941-241d28dc-655a-44f9-affe-50f37c682373.png)

- Should the color of the text into the input be editable?
- Should `Remove item` text have the color of the text or the color of the link?

#### Empty Cart

![image](https://user-images.githubusercontent.com/4463174/147126210-45a899ff-fa8d-4778-a86b-12752d33e902.png)

- Should the text into the button have the color of the text or the color of the link?


### Changelog

> Add text and link color support for Mini Cart Contents Block
